lib/Driver/Compilation.cpp

```diff
@@ -393,6 +415,23 @@ namespace driver {
         DriverTimers[FinishedCmd]->stopTimer();
       }
 
+      if (BatchJobs.count(FinishedCmd) != 0) {
+        if (Comp.ShowJobLifecycle)
+          llvm::outs() << "Batch job finished: "
+                       << LogJob(FinishedCmd) << "\n";
+        auto const *B = static_cast<const BatchJob *>(FinishedCmd);
```

Comment:
I just realized Job needs a virtual destructor if BatchJob is going to contain a SmallVector.

Reply:
Whoops, fixed!